### DISCUSSION TITLE
Replacement Z Couplers

### DISCUSSION THREAD

**Discussion by armishmajerj** [bassamanator/everything-sovol-sv06#7] [Q&A] [answered]
In my quest of diagnosing my unleveling bed I discovered one of the z coupler screws/screwholes were too tight and stripped. Whilst homing one side of the Z axis went up while the other stayed. I quickly hit the power switch to stop the printer and believe I stopped more damage from occurring but am in need of replacement Z couplers. I figured I may as well start the discussion and link to parts I end up using or are compatible. I wanted to ask though since this is my first time in this situation what couplers I would need. I believe from looking at the files from Sovol the diameter of the rods are 8mm. When looking at replacements I see there are other kinds such as flexible ones that I don't know if they would be able to be used or if I need to stick to rigid ones like the machine came with. Also I assume the size on the coupler I just need to make sure the diameter of the rod is somewhere in between what the site lists correct? So like these ones may work? https://www.amazon.com/Befenybay-Aluminium-Flexible-Coupling-Connector/dp/B07RMZCLZ3/ref=sr_1_2?crid=38MW1GFRXD1K4&keywords=sv06+z+coupler&qid=1677548783&sprefix=sv06+z+couple%2Caps%2C98&sr=8-2

**Comment by bassamanator** [comment, marked as answer]
Sorry to hear that but looks like you were in time to save the printer. These things happen, makes you a better 3d printer user ultimately.

The couplers you linked to are known as 'Spider' couplers. I quite like them, and have spares that I will eventually add to my SV06. The dimensions are also correct, 5mm stepper motor shaft (*I have not measured this myself, but I'm 99.99% sure this is the case because it's kind of a standard for 3d printers*), and 8mm lead screws. The only thing to watch out is that these couplers, 5-8 in this case,  can come in many diameters and legths. You're fine with D20, but larger diameters could end up touching the lead rods, which would be a problem.

Feel free to create a pull request to add this information to the repo, or I'll get to it eventually.

PS these couplers can be had for about $2-3 each from Aliexpress, but I understand that that wait can often be quite painful.

**Comment by armishmajerj** [comment]
Looking further at the couplers/screws it seems that only the one screw has become stripped and will not tighten the couplers anymore top or bottom. I will look at ordering some of these spider couplers from aliexpress then 👍 I just need to make a trip to Home Depot tomorrow to find some screws to replace on the coupler. I believe it is a 2.5mm hex head but not sure on length yet. Will bust out calipers and measure to be sure and post once I find out

**Comment by bassamanator** [comment]
I would take the good one with me, they might have a pitch gauge to measure the pitch and what not, though usually home depot is bad with smaller sized machine screws. Feel free to post what you find here, so that it can be added to the repo.
